yew-macro/src/html_tree/html_prop.rs

```diff
@@ -19,6 +20,11 @@ impl PeekValue<()> for HtmlProp {
 impl Parse for HtmlProp {
     fn parse(input: ParseStream) -> ParseResult<Self> {
         let label = input.parse::<HtmlPropLabel>()?;
+        let question_mark = if input.peek(Token![?]) {
+            Some(input.parse::<Token![?]>()?)
```

Comment:
I feel like the variable name `question_mark` is pretty self-explanatory already.
```suggestion
            Some(input.parse()?)
```